### DISCUSSION TITLE
mosquitto doesn't seem to work

### DISCUSSION THREAD

**Discussion by thany** [simonsickle/unraid-templates#7] [Q&A / Support]
There's not a whole lot to go on, unfortunately. I just installed the docker, and accepted the default port 1883. Network is bridged, as you know is the default, and I know that part works fine. I also know port 1883 wasn't already used for something.

The result? I cannot access the console or the logs. Whenever I try opening either, a new window opens, and closes right away. As if both are somehow not accessible? It does start "successfully" if I have to believe the status icon Unraid provides. It goes without syaing any MQTT client is unable to access it.

I'm on Unraid 6.12.6 and I just installed mosquitto for the very first time. I didn't do any form of preparation before installing it, or manual configuration after it "started". So in essence, it's a totally vanilla installation.

I reckon it should probably at least do *something* in this scenario, right?

**Comment by 0xE1** [comment]
Create an empty `mosquitto.conf` file in config/ folder, the process keeps restarting itself looking for that file which is why can't open console or logs.
`Error: Unable to open config file /mosquitto/config/mosquitto.conf.`

**Reply by thany** [reply]
Why can't such file be created automatically?

Edit: or even ignored if it doesn't exist.
Either way, this is a manual step that could be automated very easily.

**Reply by qs-data** [reply]
Thanks!  This worked for me!

Never really checked on my mosquitto docker, just got it running when I got my weather station as I thought it was needed.  I think it's been doing this the whole time.  Now I'm not sure that I even need mosquitto if things have been working this long without it.

**Comment by bucky2780** [comment]
some more strange behaviour as the default user/group is 1883 ? What is with that ?
Does this template actually work ? Can I suggest you withdraw it, until such time it is workable so as not to confuse users trying this out ?
